apps/public-reference/pages/applications/review/summary.tsx

```diff
@@ -4,21 +4,78 @@ Display a summary of application fields with edit links per section
 */
 import Link from "next/link"
 import Router from "next/router"
-import { Button, FormCard, ProgressNav } from "@bloom-housing/ui-components"
+import { Address } from "@bloom-housing/core"
+import { Button, FormCard, ProgressNav, MultiLineAddress, t } from "@bloom-housing/ui-components"
 import FormsLayout from "../../../layouts/forms"
 import { useForm } from "react-hook-form"
 import { AppSubmissionContext } from "../../../lib/AppSubmissionContext"
 import ApplicationConductor from "../../../lib/ApplicationConductor"
 import { useContext } from "react"
 
+const EditLink = (props: { href: string }) => (
+  <div className="float-right">
+    <Link href={props.href}>
+      <a className="button is-unstyled is-borderless uppercase text-base my-2 text-primary-dark">
+        Edit
+      </a>
+    </Link>
+  </div>
+)
+
+const ReviewItem = (props: { label: string; sublabel?: string; children?: any }) => (
+  <p className="mb-2">
+    <span className="text-gray-700">{props.label}</span>
+    {props.children && (
+      <>
+        <br />
+        <span className="field-label--caps font-alt-sans text-base text-black">
+          {props.children}
+        </span>
+      </>
+    )}
+    {props.sublabel && (
+      <>
+        <br />
+        <span className="-mt-3 block text-base">{props.sublabel}</span>
+      </>
+    )}
+  </p>
+)
+
+const reformatAddress = (address: any) => {
```

Comment:
Why isn't this typed? At the very least, we need it to be `object` or `Record<string, any>`, since we're pulling keys from it, but it seems like it'd be very easy to specify what this object looks like. If we just throw `any` all over the place, Typescript doesn't actually give us anything other than extra boilerplate...

Reply:
Ah, I forgot to mention that we have a slight mismatch between our addresses in applications and the addresses used for listing display from past work, so we'll need to align the two data structures. Probably should be done as part of the TS data modeling for applications.